Jenkinsfile

```diff
@@ -181,9 +183,7 @@ stage('unit-tests') {
 
 stage('publish') {
   node('docker') {
-    publishReport('linux')
```

Comment:
Building with coverage enabled greatly increases build time, and because we don't try to test any of the non-sync fallback code paths, the non-sync coverage reports don't tell us anything useful that isn't already covered by the sync-enabled coverage reports.